aws/etl_lambdas/policies.tf

```diff
@@ -56,21 +56,23 @@ data "aws_iam_policy_document" "etl_policies" {
 
   statement {
 
-    effect = "Allow"
+    effect = "Allow lambdas to write to logs"
```

Comment:
This should be `effect="Allow"` and then you could add a `sid="Allow lambdas to write to logs"`.

Reply:
nice catch, I thought I had not finished the sid...

Reply:
Also another one to add to OPA checks

Reply:
Updated issue https://github.com/cds-snc/site-reliability-engineering/issues/127